### ISSUE TITLE
Update ID format of Baseline tests

### ISSUE THREAD

**Issue by kengdoj** [atbcb/ICTTestingBaseline#435]
2 tasks:

1.  To add distinction from WCAG SCs, letter the tests in each Baseline category. 

    -    For example: Baseline Test ID 1.1-1 would become 1.A-1. 

3. Update ID numbers if out of numerical sequence. 

     - For example: the first test instruction ID should be numbered 2.
